.coderabbit.yaml

```
@@ -0,0 +1,182 @@
+
+# Enable early access features
+early_access: true
+
+# Enable free tier features
+enable_free_tier: true
+
+# Review configuration
+reviews:
+  # Use assertive profile for more thorough feedback
+  profile: "assertive"
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)